.prettierrc

```diff
@@ -1,9 +1,11 @@
 {
-  "printWidth": 100,
-  "semi": false,
```

Comment:
Due to how typescript works, we currently have some ugly things going on because this isn't set to true.

`;(GuardedProfile as any).getInitialProps` for example as we need the semicolon to tell the browser it's not a function.

Reply:
I like to have it without. But there are some rare cases it can lead to faults. I think @LukasSkywalker will be all for it ;) So I could agree.

Reply:
Oooooh yeah. Michael knows me too well. I prefer the explicit semicolon.